File: retrolambda/src/main/java/com/github/daneko/android/retrolambda/Extensions.java
package com.github.daneko.android.retrolambda;

import android.view.View;

import android.support.design.widget.Snackbar;

/**
 *
 */
public class Extensions {
    public static class ViewExtensions {
        public static Snackbar snack(View view, String text) {
            return Snackbar.make(view, text, Snackbar.LENGTH_LONG).setAction("Action", null);
        }
    }
}

File: kotlin/src/main/kotlin/com/github/daneko/android/kotlin/Extensions.kt
package com.github.daneko.android.kotlin

import android.support.design.widget.*
import android.view.*

/**
 *
 */
fun View.snack(text: String): Snackbar =
        Snackbar.make(this, text, Snackbar.LENGTH_LONG).
                setAction("Action", null)
